src/components/icon-tile/docs.mdx

```diff
@@ -0,0 +1,30 @@
+---
+componentName: 'IconTile'
+---
+
+IconTiles are used to represent objects and resources,
+either unrelated, or directly related to a product.
+
+Renders an icon in a brand-colored tile. Note that this IconTile does
+not fully implement [the IconTile component documented in the
+HashiCorp Design System](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components-%5BWIP%5D?node-id=1377%3A11992), as it does not support "secondary" icons.
+
+<LiveComponent>{`
+    <IconTile size="large"><svg id="flight-smile-16" viewBox="0 0 16 16" width="100%" height="100%"><g><path d="M8 12c-1.01 0-1.782-.504-2.267-.945a4.72 4.72 0 01-.564-.614 3.31 3.31 0 01-.212-.305.75.75 0 011.284-.775 3.214 3.214 0 00.5.584c.341.31.769.555 1.259.555.49 0 .918-.246 1.258-.555a3.214 3.214 0 00.5-.584.75.75 0 011.285.775l-.212.305c-.128.167-.317.39-.564.614C9.782 11.495 9.01 12 8 12zM5 6a1 1 0 011-1h.007a1 1 0 010 2H6a1 1 0 01-1-1zM10 5a1 1 0 100 2h.007a1 1 0 100-2H10z"></path><path fill-rule="evenodd" d="M8 0a8 8 0 100 16A8 8 0 008 0zM1.5 8a6.5 6.5 0 1113 0 6.5 6.5 0 01-13 0z" clip-rule="evenodd"></path></g></svg></IconTile>
+`}</LiveComponent>
+
+## Examples
+
+Due to limitations in our ability to import external packages into Swingset documentation pages, it's unfortunately not practical to fully demonstrate the use of `@hashicorp/flight-icons` with `IconTile`. An example of use should look something like:
```

Comment:
Feels a bit harder to show useful examples of `IconTile`, since I think I'd have to inline the SVG as in the `LiveComponent`. Have opted for a code block example here.

> The way this is rendering isn't great IMO - making me think about getting some better default content styles in place for Swingset. But that definitely feels out of scope for this PR.

Reply:
> making me think about getting some better default content styles in place for Swingset

I've been feeling similarly! Maybe we can get some table styles in too? 😊 (agree, not in the scope of this PR)